testutil/gist.go

```diff
@@ -25,7 +25,7 @@ const (
 func (g *InMemoryGist) Upload(content string) (string, error) {
 	if g.FailNext {
 		g.FailNext = false
-		return "", errors.New("Gist upload failed")
+		return "", errors.New("gist upload failed")
```

Comment:
You can tell that I started this project by using a linter, and gave up eventually